infra/chromadb_compaction/tests/test_smoke.py

```
@@ -0,0 +1,168 @@
+
+    def test_field_change_dataclass(self):
+        """Test FieldChange dataclass basic functionality."""
+        from ..lambdas.stream_processor import FieldChange
+
+        change = FieldChange(old="old_value", new="new_value")
+
+        assert change.old == "old_value"
+        assert change.new == "new_value"
+
+        # Test that frozen dataclass prevents modification
+        with pytest.raises(Exception):  # Should be frozen
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)